test/run

```diff
@@ -2,6 +2,14 @@
 
 set -eu
 
+test_ruff() {
+    command -v ruff >/dev/null || echo 'no ruff installed' && return 0
```

Comment:
That now returns if ruff is available. You need `|| { echo 'no ruff installed'; return 0; }`